packages/core/src/indexing/postgres/migrations/1-create-model-table.ts

```diff
@@ -72,3 +74,25 @@ export async function createModelTable(
     )
   })
 }
+
+export async function createConfigTable(dataSource: Knex, tableName: string) {
+  switch (tableName) {
+    case INDEXED_MODEL_CONFIG_TABLE_NAME:
+      await dataSource.schema.createTable(tableName, function (table) {
+        // create model indexing configuration table
+        table.string('model', 1024).unique().notNullable().primary()
+        table.boolean('is_indexed').notNullable().defaultTo(true)
+        table.dateTime('created_at').notNullable().defaultTo(dataSource.fn.now())
+        table.dateTime('updated_at').notNullable().defaultTo(dataSource.fn.now())
+        table.string('updated_by', 1024).notNullable()
+
+        table.index(['is_indexed'], `idx_ceramic_is_indexed`, {
```

Comment:
I guess making the `model` column "primary" means it already has an index created by default, and that's why we don't need to define one here?